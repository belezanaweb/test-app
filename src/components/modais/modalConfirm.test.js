import 'jest-styled-components';
// do not forget the /native at the end of the line.
import React from 'react';
import renderer from 'react-test-renderer';
import ModalConfirm from './modalConfirmPurchase';

describe('Component Modal Confirma', () => {
	test('test Snapshot', () => {
		const tree = renderer.create(<ModalConfirm />).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
