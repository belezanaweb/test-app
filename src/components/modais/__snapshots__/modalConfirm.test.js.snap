// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Modal Confirma test Snapshot 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "backgroundColor": "#ffffff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
>
  <RCTScrollView
    style={
      Array [
        Object {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
        },
      ]
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "paddingBottom": 0,
              "paddingLeft": 20,
              "paddingRight": 20,
              "paddingTop": 0,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
                "marginTop": 50,
                "width": "100%",
              },
            ]
          }
        >
          <Image
            resizeMode="cover"
            source={
              Object {
                "testUri": "../../../src/assets/images/icon_success.png",
              }
            }
            style={
              Object {
                "height": 163,
                "width": 163,
              }
            }
          />
        </View>
        <Text
          style={
            Array [
              Object {
                "fontFamily": "Roboto-Medium",
                "fontSize": 14,
                "lineHeight": 16,
                "textAlign": "center",
                "textTransform": "uppercase",
                "width": "100%",
              },
            ]
          }
        >
          Seu produto foi adicionado com sucesso รก sacola
        </Text>
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
